### ISSUE TITLE
Character walk/animation speed makes them look like they're skating a little bit

### ISSUE THREAD

**Issue by agmathew** [quiver-dev/template-beat-em-up#108]
Something about the walk speed or animation FPS makes both the player and enemy characters look like they're skating a little bit. Not sure if there's a way to make this look a little smoother.

**Comment by eh-jogos** [comment]
@agmathew just pushed some tweaks to character speed and walk animation speeds (bc98f7bf0305b3bd34209bfe1cbb7ffb444eb1d2). Please take a look if it's smoother, I think my eye is too biased to evaluate this.

**Comment by agmathew** [comment]
For Chad, this looks better to me. Did you apply the changes to Sarge? If not, I think it could use similar tweaks and he still appears like he's skating.

**Comment by eh-jogos** [comment]
Yes, I had applied the changes to all characters, Sarge and TaxMan as well. I'll tweak Sarge more then!

**Comment by eh-jogos** [comment]
@agmathew I just pushed to main commit 7ffaef12dd9918789018b7203f6d79a11001183a that tweaks sarge's walk animations further. Do you think they look better?
